.rubocop.yml

```diff
@@ -191,4 +191,4 @@ Rails/I18nLocaleTexts:
     - spec/**/*.rb
 
 RSpec/VerifiedDoubleReference:
-  Enabled: false
```

Comment:
The cop is for enforcing a consistency on verified double referencing, it's either referenced as a class or referenced as a string that represents what class it should be. For all instances where there's an instance double created for ResultSet in tests, it fails when referencing the class because the class doesn't exist. I'm not actually sure what is being referenced here to be honest - ResultSet seems to be a catch all for a set of results that comes from JSON API Client. Assuming this cop was caught out due to a rubocop update as I haven't changed a lot of the tests that fail when this line is disabled